python-sdk/python_sdk_custom_docker_demo/build_docker.sh

```diff
@@ -0,0 +1,3 @@
+docker build -t raymondlo84/disco_python_sdk -f disco.dockerfile .
+docker run -it raymondlo84/disco_python_sdk:latest
```

Comment:
This will also run the docker and not just build it. Was that your intention?

Reply:
Yeah, but i would remove the run line from the script. If it's a build script it should be that and only that.
Otherwise, change the name to build_and_run.sh, but i think it's not the purpose here.